src/phoenix/experimental/callbacks/langchain_tracer.py

```diff
@@ -68,10 +76,50 @@ def _prompt_template(serialized: Dict[str, Any]) -> Iterator[Tuple[str, Any]]:
                 yield LLM_PROMPT_TEMPLATE_VARIABLES, kwargs.get("input_variables", [])
                 yield LLM_PROMPT_TEMPLATE_VERSION, "unknown"
                 break
-        except TypeError:
+        except (AttributeError, KeyError, TypeError):
             continue
 
 
+def _invocation_parameters(extra: Dict[str, Any]) -> Iterator[Tuple[str, str]]:
+    """Yields invocation parameters if present."""
+    yield LLM_INVOCATION_PARAMETERS, json.dumps(extra.get("invocation_params", {}))
+
+
+def _model_name(extra: Dict[str, Any]) -> Iterator[Tuple[str, str]]:
```

Comment:
```suggestion
def _model_name(run_extra: Dict[str, Any]) -> Iterator[Tuple[str, str]]:
```
nit on the parameter name - it's more clear if the parameter means something to the reader here.

Reply:
Addressed in f6b077c